eStore/Scenes/Theme/ThemeSettingsView.swift

```diff
@@ -6,4 +6,145 @@
 //  Copyright © 2019 Vladislav Kondrashkov. All rights reserved.
 //
 
-import Foundation
+import UIKit
+
+final class ThemeSettingsViewImpl: UIViewController {
+    var presenter: ThemeSettingsPresenter!
+    var theme: Theme!
+
+    private var themeSectionedMenu: SectionedMenuView!
+    private var colorPickerItems: [TintColorType] = []
+
+    override func loadView() {
+        view = UIView()
+
+        themeSectionedMenu = SectionedMenuView(frame: .zero, style: .grouped)
+        view.addSubview(themeSectionedMenu)
+        themeSectionedMenu.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "Theme settings"
+        navigationItem.largeTitleDisplayMode = .never
+
+        themeSectionedMenu.tableFooterView = UIView() // Is needed to remove unnecessary separators
```

Comment:
I believe it is known behavior, no need to specify an explanation everywhere... but if you want who am I complain 🙂